starters/next-drupal-starter/components/preview-ribbon.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+
+export default function PreviewRibbion() {
```

Comment:
This is great! Eventually we should move this to the `nextjs-kit`, but given only Drupal supports preview at the moment, it's fine to keep it here.

Small typo on the function name:
```suggestion
export default function PreviewRibbon() {
```